### ISSUE TITLE
some warnings in Atom but no warning in console

### ISSUE THREAD

**Issue by oupala** [wooorm/linter-remark#13]
I'm using Atom, remark-lint and linter-remark to lint my markdown files.

I have some warnings on Atom console:

> Cell should be padded (remark-lint:table-cell-padding)

But no warning on my terminal:

```bash
$ remark my-file.md
[ ... some content ... ]
my-file.md: no issues found
```

The table to pad is perfectly padded (so remark on the console is right).

Can you help me fix this problem?

**Comment by wooorm** [comment]
Please provide more info on applicable remarkrc files and versions of packages.
I’m guessing you have an up to date `linter-remark` but your remark packages are out of date?

**Comment by oupala** [comment]
Here is the markdown file:

```md
## imprimante

| nom         | localisation | serveur et file d'impression                                | modèle                           | driver                                                                                            |
| ----------- | ------------ | ----------------------------------------------------------- | -------------------------------- | ------------------------------------------------------------------------------------------------- |
| w7-aaaaaaaa | bâtiment A   | lpd://aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa | Canon ImageRunner Advance C5030i | [Canon iR-ADV C5030/5035 PPD](https://www.openprinting.org/printer/Canon/Canon-iR-ADV_C5030_5035) |
| w7-bbbbbbbb | bâtiment B   | lpd://bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb             | Canon ImageRunner Advance C2220i | [Canon iR-ADV C2220/2230 PPD](https://www.openprinting.org/printer/Canon/Canon-iR-ADV_C2220_2230) |
| w7-cccccccc | bâtiment B   | lpd://ccccccccccccccccccccccccccccccccccccccc               | Canon ImageRunner Advance  4235i | [Canon iR-ADV 4225/4235 PPD](https://www.openprinting.org/printer/Canon/Canon-iR-ADV_4225_4235)   |
```

The remark config file:

```js
{
  "plugins": [
    "frontmatter",

    ["lint-file-extension", "md"],
    "lint-no-file-name-mixed-case",
    "lint-no-file-name-articles",
    "lint-no-file-name-irregular-characters",
    "lint-no-file-name-consecutive-dashes",
    "lint-no-file-name-outer-dashes",
    "lint-no-consecutive-blank-lines",
    ["lint-maximum-line-length", 80],
    "lint-no-shell-dollars",
    "lint-hard-break-spaces",
    ["lint-heading-style", "atx"],
    "lint-heading-increment",
    "lint-no-duplicate-headings",
    "lint-no-multiple-toplevel-headings",
    "lint-maximum-heading-length",
    ["lint-no-heading-punctuation", ":."],
    ["lint-blockquote-indentation", 2],
    "lint-no-blockquote-without-marker",
    ["lint-unordered-list-marker-style", "-"],
    ["lint-ordered-list-marker-style", "."],
    ["lint-ordered-list-marker-value", "one"],
    ["lint-list-item-indent", "mixed"],
    "lint-list-item-content-indent",
    "lint-list-item-spacing",
    ["lint-code-block-style", "fenced"],
    ["lint-fenced-code-flag", {"allowEmpty": false}],
    ["lint-fenced-code-marker", "`"],
    ["lint-rule-style", "---"],
    "lint-no-table-indentation",
    "lint-table-pipes",
    "lint-table-pipe-alignment",
    ["lint-table-cell-padding", "padded"],
    "lint-no-inline-padding",
    "lint-no-shortcut-reference-image",
    "lint-no-shortcut-reference-link",
    "lint-final-definition",
    "lint-definition-case",
    "lint-definition-spacing",
    ["lint-link-title-style", "\""],
    ["lint-strong-marker", "*"],
    ["lint-emphasis-marker", "*"],
    "lint-no-emphasis-as-heading",
    "lint-no-literal-urls",
    "lint-no-auto-link-without-protocol",

    "lint-final-newline",
    "lint-list-item-bullet-indent",
    ["lint-list-item-indent", "tab-size"],
    "lint-no-auto-link-without-protocol",
    "lint-no-blockquote-without-marker",
    "lint-no-literal-urls",
    ["lint-ordered-list-marker-style", "."],
    "lint-hard-break-spaces",
    "lint-no-duplicate-definitions",
    "lint-no-heading-content-indent",
    "lint-no-inline-padding",
    "lint-no-shortcut-reference-image",
    "lint-no-shortcut-reference-link",
    "lint-no-undefined-references",
    "lint-no-unused-definitions",

    ["lint-list-item-indent", "space"],
    ["lint-maximum-line-length", false],
    ["lint-no-multiple-toplevel-headings", false]
  ]
}
```

**Comment by wooorm** [comment]
I also asked about versions and had a suggestion: did you try it?

**Comment by oupala** [comment]
I was checking the version of `linter-remark` I was using, and I just discovered that I was using `linter-markdown`.

I must have been confusing both packages. Can you tell me the differences between the two packages?

**Comment by wooorm** [comment]
One has many open issues and is unmaintained (linter-markdown), the other (this one) isn’t

**Comment by oupala** [comment]
Thanks, I switched to `linter-remark` and I do not get any difference between Atom and the console. Thanks!

**Comment by wooorm** [comment]
Perfect!
